src/ServiceBus.AttachmentPlugin/ServiceBus.AttachmentPlugin.csproj

```diff
@@ -24,6 +27,12 @@
     <AssemblyOriginatorKeyFile>..\key.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Content Remove="../../images/project-icon.png" />
+    <Content Include="../../images/project-icon.png" Pack="true" PackagePath=""/>
+    <None Include="..\..\$(PackageLicenseFile)" Condition="Exists('..\..\$(PackageLicenseFile)')" Pack="true" PackagePath="$(PackageLicenseFile)" Visible="false" />
```

Comment:
should not need this with the license expression

Reply:
Not entirely sure.
If you use `<PackageLicenseExpression>MIT</PackageLicenseExpression>`, the license you get is linked to https://licenses.nuget.org/MIT. That one is missing the proper copyright.